File: PHP/account/Account.php
<?php 
    class Account{
    private $id; 
    private $name;
    private $document;
    private $email;
    private $password;
    
    public function __construct($name, $document)
    {
        $this->name = $name;
        $this->document = $document;
    }

    public function getId(){
        return $this->id;
    }
    public function setName($name){
        $this->name = $name;
    }
    public function getName(){
        return $this->name;
    }

    public function setDocument($document){
        $this->document = $document;
    }
    public function getDocument(){
        return $this->document;
    }

    public function setEmail($email){
        $this->email = $email;
    }
    public function getEmail(){
        return $this->email;
    }
    public function setPassword($pass){
        $this->password = $pass;
    }
    public function getPassword(){
        return $this->password;
    }

    }
?>

File: Python/account/Account.py
class Account:
    __id = int
    __name = str
    __document = str
    __email = str
    __password = str

    def __init__(self,name,document):
        self.name=name
        self.document=document

    def getId(self):
        return self.__id
    
    def setName(self,name):
        self.__name=name
    
    def getName(self):
        return self.__name
     
    def setDocument(self,name):
        self.__document=name 

    def getDocument(self):
        return self.__document
    
    def setEmail(self,email):
        self.__email = email
    
    def getEmail(self):
        return self.__email

    def setPassword(self,password):
        self.__password=password
    
    def getPassword(self):
        return self.__password

    def printData(self):
        return "name: "+self.__name+" document: "+self.__document



    